apps/web/src/__tests__/routes/vibes/$vibeId.test.tsx

```
@@ -158,21 +159,31 @@ vi.mock('@/components/emoji-rating-popover', () => ({
       });
     };
 
+    const handleEmojiSelect = (emoji: string) => {
+      setSelectedEmoji(emoji);
+      // Force immediate re-render
+      setForceRender(prev => prev + 1);
```

ESLint

[error] Replace `prev` with `(prev)`

(prettier/prettier)